rmgpy/rmg/inputTest.py

```diff
@@ -28,10 +28,11 @@
 #                                                                             #
 ###############################################################################
 
-import unittest 
+import unittest
 
-from rmgpy.rmg.main import RMG
 from rmgpy.rmg import input as inp
```

Comment:
What would be the difference between this and `import rmgpy.rmg.input as inp`?

Reply:
After doing some searching and some playing around in python, there is no difference between these two statements, and it is purely up to us as to which one we prefer. It is worth noting that if it wasn't for the `as inp` statements in each that there would be a slight difference, as `import rmgpy.rmg.input` would put rmgpy.rmg into the namespace, whereas `from rmgpy.rmg import input` would not. There is an even more interesting but subtle effect if we were trying to import a module level variable (see the second from the top response [here](https://stackoverflow.com/questions/710551/use-import-module-or-from-module-import?noredirect=1&lq=1)), but this does not apply in this case.

Given this, which one do we prefer?

Reply:
Good to know!

I have a slight preference for `import rmgpy.rmg.input as inp` since it's cleaner and seems a bit more straightforward. I always figured that from imports were mainly if you wanted something from within a file. Up to you whether or not to change it though.